### DISCUSSION TITLE
Why my building is black in the render progress?

### DISCUSSION THREAD

**Discussion by likehbbfoe** [NVlabs/sionna#642] [Q&A] [answered]
Hello everyone:
I get some problems, When I using Sionna.
<img width="217" alt="屏幕截图 2024-10-30 133607" src="https://github.com/user-attachments/assets/29813ffd-fcff-4ef2-aa90-f57374ec2c71">
the image is my preview scence.
But when I get the render image, my building is all becoming black!
like this
![scene](https://github.com/user-attachments/assets/bd8015b7-847c-47e4-ab54-3e0b2b70f078)
What could I do for this. Any heip is appreciated

**Comment by merlinND** [comment]
Hello @likehbbfoe,

1. How was this scene created?
2. Do the buildings still look black if you render it without the coverage map and radio devices?
3. Could you please share the full scene (XML *and* mesh files) so that we can try to reproduce the issue locally.

**Reply by likehbbfoe** [reply]
OK.
First, the scene is created by Blender using openstreet-map, also I use the mistuba to export the files.
Second, I try to render the scene without the the coverage map and radio devices. But it do not work.
![black](https://github.com/user-attachments/assets/3cd784b5-b6b7-43ee-893f-686424380677)
[new.zip](https://github.com/user-attachments/files/17599832/new.zip)
The scene is shared. Could you please help me see what the problem is?

**Reply by merlinND** [reply, marked as answer]
I think I see what happened. Your XML file does not contain any emitter (light source), so the rendering appears black. Usually in Blender there's an environment emitter by default. You can also see it in the built-in Sionna scenes.
The scene should become visible again after adding e.g. this emitter to your XML file:

```xml
<!-- Emitters -->
	<emitter type="constant" id="World">
		<rgb value="1.000000 1.000000 1.000000" name="radiance"/>
	</emitter>
```

**Reply by likehbbfoe** [reply]
OK, it works for my scence. Really thank you so much for your help.
